sale_invoice_group_method/README.rst

```diff
@@ -0,0 +1,102 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License LGPL-3
+
+=========================
+Sale Invoice Group Method
+=========================
+
+This module allows you to combine several Sales Orders into a single invoice,
+if they meet the split criteria defined by the 'Invoice Group Method'.
```

Comment:
One minor point - why does it refer to split criteria?  Surely its merge or group criteria - yes I know its not part of the port but I noticed it.